src/shared/api/client.ts

```diff
@@ -37,11 +37,15 @@ export const implApi = ({ externalCall }: ImplApiProps) => {
     p: InternalCallParams & { token: string },
   ) => internalCall<R | ErrorResponse>(p);
 
-  const callWithoutToken = <R extends ResponseNecessary>(
+  const callWithOptionalToken = <R extends ResponseNecessary>(
     p: InternalCallParams & { token?: string },
   ) => internalCall<R | ErrorResponse>(p);
```

Comment:
callWithOptionalToken에서는 token?:string이 맞는데요, callWithoutToken에서는 token?: never이 맞는 거 같습니다.
```suggestion
  const callWithOptionalToken = <R extends ResponseNecessary>(
    p: InternalCallParams & { token?: string },
  ) => internalCall<R | ErrorResponse>(p);
  
   const callWithoutlToken = <R extends ResponseNecessary>(
    p: InternalCallParams & { token?: never },
  ) => internalCall<R | ErrorResponse>(p);
```

Reply:
제가 타입을 잘못 썼었네요 😂 고쳐주시면 감사하겠습니다..